README.md

```diff
@@ -0,0 +1,157 @@
+# `type-proxy`
+
+Copyright 2021 Vivi International (MIT)
```

Comment:
I'd recommend moving this to a separate LICENSE section at the end of the README, e.g.:
```
## License

This project has been published under the MIT License.

See the LICENSE file for more information.
```